<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReittiDialogi</class>
 <widget class="QWidget" name="ReittiDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reittien suunnittelu</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,4,0,1">
   <item>
    <widget class="QListWidget" name="reittiLista"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="lisaaReittiNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/lisaa.png</normaloff>:/r/pic/lisaa.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kopioiNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/kopioi.png</normaloff>:/r/pic/kopioi.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaReittiNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/poista.png</normaloff>:/r/pic/poista.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="tallennusVaroitus">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ratapiha-kauko.qrc">:/r/pic/varojunaa.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="reittiView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="lisaaPysaysNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/lisaa.png</normaloff>:/r/pic/lisaa.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaPysaysNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/poista.png</normaloff>:/r/pic/poista.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="kellostaNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/kellolla.png</normaloff>:/r/pic/kellolla.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pyyhiajatNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/eiaikoja.png</normaloff>:/r/pic/eiaikoja.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="hyvaksyNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/hyvaksy.png</normaloff>:/r/pic/hyvaksy.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hylkaaNappi">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/peru.png</normaloff>:/r/pic/peru.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="liikennepaikkaLista">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ratapiha-kauko.qrc"/>
 </resources>
 <connections/>
</ui>
